pkg/api/validation/schema.go

```diff
@@ -311,6 +311,9 @@ func (s *SwaggerSchema) validateField(value interface{}, fieldName, fieldType st
 		return s.ValidateObject(value, fieldName, fieldType)
 	}
 	allErrs := []error{}
+	if reflect.TypeOf(value) == nil {
```

Comment:
Hm, I would expect IsNil to work. Anyway. Can you also add a test in hack/test-cmd.sh that tests that the panic is fixed?